Update getting started (#2233)
grafana/mimir: docs/sources/operators-guide/getting-started-helm-charts/_index.md

1. Start Grafana by using Docker:

    ```bash
    docker run --rm --name=grafana --network=host grafana/grafana
    ```

1. In a browser, go to the Grafana server at [http://localhost:3000](http://localhost:3000).
1. Sign in using the default username `admin` and password `admin`.
1. Navigate to Configuration -> Data sources on the left.
1. Configure a new Prometheus data source to query the local Grafana Mimir server using the following settings:

      | Field | Value                              |
      | ----- | ---------------------------------- |
      | Name  | Mimir                              |
      | URL   | http://\<ingress-host\>/prometheus |

    To add a data source, refer to [Add a data source](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/) in the Grafana documentation.

 1. Verify success

    When you have completed the tasks so far in this getting started guide, you can query metrics in [Grafana Explore](https://grafana.com/docs/grafana/latest/explore/)
    as well as create dashboard panels using the newly configured Grafana Mimir data source.

## Set up meta monitoring

> Grafana Mimir meta monitoring collects metrics and or logs about Grafana Mimir itself. It's primary purpose is to send meta monitoring information to some external receiver, for example a [free tier Grafana Metrics account](https://grafana.com/metrics/). In this example we'll use it to scrape metrics about Grafana Mimir itself and then write those metrics into the same Grafana Mimir.

Reviewer: ```suggestion
Grafana Mimir meta monitoring collects metrics and or logs about Grafana Mimir itself. Its primary purpose is to send meta-monitoring information to some external receiver, for example a [free tier Grafana Metrics account](https://grafana.com/metrics/). In the example that follows, it scrapes metrics about Grafana Mimir itself and then writes those metrics to the same Grafana Mimir instance.
```
